nilai-attestation/src/nilai_attestation/attestation/__init__.py

```diff
@@ -0,0 +1,43 @@
+from functools import lru_cache
+from nilai_common import AttestationReport, Nonce
+
+from nilai_attestation.attestation.sev.sev import sev
+from nilai_attestation.attestation.nvtrust.nv_attester import nv_attest
+from nilai_attestation.attestation.nvtrust.nv_verifier import verify_attestation
+from nilai_common.logger import setup_logger
+
+logger = setup_logger(__name__)
+
+
+@lru_cache(maxsize=1)
+def load_sev_library() -> bool:
+    """Load the SEV library"""
+    return sev.init()
```

Comment:
This seems a bit hacky. I understand you are trying to load it only once. Have you tried to initialize it globally on import? It should be called only once.  And you are not checking if the init failed swallowing the error and delaying it to the execution of the attestation.
```suggestion
if not sev.init():
    raise Exception("Failed to load SEV")
```

Reply:
At the moment, I aim at allowing nilAI to run on SEV and non-SEV hosts. If `/dev/sev-guest` is not available, the idea is that it continues to work, though with fake attestations. That's why we need to make sure an exception is not raised here.